frontend/ui/data/koptoptions.lua

```diff
@@ -76,6 +76,136 @@ In 'semi-auto' and 'manual' modes, you may need to define areas once on an odd p
+                values = {2, 1, 0},
+                default_value = 2,
+                show_func = function(config) return config and config.zoom_mode_genus > 2 end,
+                event = "DefineZoom",
+                args = {"full", "width", "height"},
+                name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[Set zoom mode.]]),
+            },
+            {
+                name = "zoom_range_number",
+                name_text = "Number",
+                buttonprogress = true,
+                fine_tune = true,
+                fine_tune_param = {value_step = 0.1},
+                values = {1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0},
+                default_pos = 2,
+                default_value = 2,
+                show_func = function(config)
+                    return config and config.zoom_mode_genus < 3 and config.zoom_mode_genus > 0
+                end,
+                event = "DefineZoom",
+                args =   {1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0},
+                labels = {1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0},
+                name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[Set zoom mode.]]),
+            },
+            {
+                name = "zoom_factor",
+                name_text = "Factor",
+                buttonprogress = true,
+                more_options = true,
+                more_options_param = {value_step = 0.1, precision = "%.1f",},
+                values = {0.7, 1.0, 1.3, 1.6, 2.0, 2.5, 3.0, 4.0},
+                default_pos = 2,
+                default_value = 1.6,
+                show_func = function(config)
+                    return config and config.zoom_mode_genus < 1
+                end,
+                event = "DefineZoom",
+                args =   {1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0},
+                labels = {1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0},
+                name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[Set zoom mode.]]),
+            },
+            {
+                name = "zoom_mode_genus",
+                name_text = _("Zoom to fit"),
+                -- toggle = {_("page"), _("content"), _("columns"), _("rows"), _("manual")},
+                item_icons = {
+                    "resources/icons/zoom.page.png",
+                    "resources/icons/zoom.content.png",
+                    "resources/icons/zoom.direction.column.png",
+                    "resources/icons/zoom.direction.row.png",
+                    "resources/icons/zoom.manual.png",
+                },
+                alternate = false,
+                values = {4, 3, 2, 1, 0},
+                default_value = 4,
+                event = "DefineZoom",
+                args = {"page", "content", "columns", "rows", "manual"},
+                name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[Set zoom mode.]]),
```

Comment:
I don't think we should bother the translators with placeholder texts like this. Please either write an actual explanation or open an issue to track it (otherwise I'll probably forget about it).

```suggestion
```

Reply:
Is it necessary to keep `help_text` for such self-documenting features?

Reply:
Probably not, or may be not right now.
I think at some point, we might want to do some little write up to explain what works with what other settings, give some sample use cases where this might be useful, in combination with which other settings...

Reply:
Definitely not if it says practically the same thing on the left. As to if it should say something and what it should say I'm not overly concerned for this specific PR. It's easy to add a quick overview of the current values or some actual  text in the future.